lib/actions/reassemble.js

```diff
@@ -12,7 +12,8 @@ async function timer(this_) {
   }
 }
 
-async function processAction(msg) {
+async function processAction(msg,cfg) {
+  const mode = cfg.mode
```

Comment:
Did you forget to check in the changes to `component.json` where you add the new dropdown?

Reply:
Yes! I didn't do the complete transfere from my test environment. 
You should be able to see it now.